minutes/meeting-20210112-7.md

```diff
@@ -0,0 +1,169 @@
+# Minutes of Meeting 7, 2021-01-12, 20:30–22:00 UTC
+
+- Acting chair: Liang Hai
+- Scribes: Liang Hai, et al.
+- Agenda: https://github.com/w3c/font-text-cg/issues/39
+
+## Resolutions
+
+None
+
+## Actions
+
+1. Liang Hai: Schedule discussion of https://github.com/unicode-org/text-rendering-tests for Meeting 8; try inviting Sascha.
+2. Norbert Lindenberg: Create an issue ticket for the interaction of font fallback and script/run itemization.
```

Comment:
Should be: Add information on the interaction between font fallbacks and script/run/cluster segmentation to issue #37.